fronts-client/src/components/EditionFeedSectionHeader.tsx

```diff
@@ -127,20 +176,23 @@ const mapStateToProps = () => {
 };
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  startConfirmPublishModal: (
+  startConfirmProofOrPublishModal: (
     title: string,
     description: ReactNode,
+    buttonText: ProofOrPublish,
     onAccept: () => void
   ) =>
     dispatch(
       startOptionsModal(
         title,
         description,
-        [{ buttonText: 'Publish', callback: onAccept }],
+        [{ buttonText, callback: onAccept }],
         noop
       )
     ),
-  publishEditionsIssue: (id: string) => dispatch(publishEditionIssue(id)),
+  proofEditionsIssue: (id: string) => dispatch(proofEditionIssue(id)),
+  publishEditionsIssue: (id: string, version?: string) =>
```

Comment:
Should `version` be optional? IIUC it's required by the server to be able to publish?

Reply:
Yes - it's not present until an issue version has been proofed.